packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -126,6 +128,14 @@ class AppContext(
     registry.postOnCreate()
   }
 
+  private fun registerInlineModulesList() {
+    try {
+      val inlineModulesList = Class.forName("inline.modules.ExpoInlineModulesList").getConstructor()
```

Comment:
Would we be able to somehow initialize this without using a string for the class name? It is always a bit scary to rely on String based types like this. I guess this is done because ExpoInlineModulesList is generated. At least we should add a comment on this line and in the generator code so that we are aware of the references?